cpprestsdk_char_traits_workaround.hpp isn't shipped in install package, preventing to build against OS SDK 3.10.0-rc5

Issue overview
--------------

Hitting a build error when trying to build OpenStudioApplication against OS SDK 3.10.0-rc5:

* https://github.com/openstudiocoalition/OpenStudioApplication/pull/809

## Current Behavior

https://github.com/openstudiocoalition/OpenStudioApplication/actions/runs/15707777753/job/44257608072?pr=809#step:18:1111

```
 In file included from /Users/runner/work/OpenStudioApplication/OpenStudioApplication/src/openstudio_lib/GeometryEditorView.cpp:51:
/Users/runner/work/OpenStudioApplication/OpenStudioApplication/build/OpenStudio-3.10.0/OpenStudio-3.10.0-rc5+e2e9b52d54-Darwin-x86_64/include/openstudio/utilities/bcl/RemoteBCL.hpp:18:12: fatal error: '../../../dependencies/cpprestsdk_char_traits_workaround.hpp' file not found
#  include "../../../dependencies/cpprestsdk_char_traits_workaround.hpp"
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

## Expected Behavior


Should work

## Steps to Reproduce

Build a C++ project that uses the OS SDK, `#include <openstudio/RemoteBCL.hpp>` into your project.

## Possible Solution

Ship that workaround file.

## Details

### Environment

Some additional details about your environment for this issue (if relevant):
 - Platform (Operating system, version): macOS only
 - Version of OpenStudio (if using an intermediate build, include SHA): 3.10.0-rc5

### Context

OpenStudioApplication

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -441,6 +441,9 @@ find_package(jsoncpp)
 find_package(fmt)
 find_package(SQLite3)
 find_package(cpprestsdk)
+# TODO: this is a workaround for #5398, it should not be there for ever
+configure_file("${PROJECT_SOURCE_DIR}/dependencies/cpprestsdk_char_traits_workaround.hpp" "${PROJECT_BINARY_DIR}/src/cpprestsdk_char_traits_workaround.hpp" COPYONLY)
+
 find_package(websocketpp)
 find_package(Boost)
 find_package(geographiclib)

--- a/src/cli/MeasureManager.hpp
+++ b/src/cli/MeasureManager.hpp
@@ -22,7 +22,7 @@
 #  pragma GCC diagnostic ignored "-Wnon-virtual-dtor"
 #endif
 #if __APPLE__
-#  include "../../dependencies/cpprestsdk_char_traits_workaround.hpp"
+#  include <cpprestsdk_char_traits_workaround.hpp>  // OpenStudio/dependencies/cpprestsdk_char_traits_workaround.hpp
 #endif
 #define _TURN_OFF_PLATFORM_STRING  // cpprestsdk has an ugly macro U() that makes fmt break...
 #include <cpprest/http_listener.h>

--- a/src/utilities/bcl/RemoteBCL.hpp
+++ b/src/utilities/bcl/RemoteBCL.hpp
@@ -15,7 +15,7 @@
 #  pragma GCC diagnostic ignored "-Wnon-virtual-dtor"
 #endif
 #if __APPLE__
-#  include "../../../dependencies/cpprestsdk_char_traits_workaround.hpp"
+#  include <cpprestsdk_char_traits_workaround.hpp>  // OpenStudio/dependencies/cpprestsdk_char_traits_workaround.hpp
 #endif
 // Macro U from cpprestsdk is clashing with (cf boost https://github.com/microsoft/cpprestsdk/issues/1214)
 #define _TURN_OFF_PLATFORM_STRING

